code/controllers/master.dm

```diff
@@ -67,6 +67,10 @@ var/CURRENT_TICKLIMIT = TICK_LIMIT_RUNNING
 	// Tell qdel() to Del() this object.
 	return QDEL_HINT_HARDDEL_NOW
 
+/datum/controller/master/proc/Shutdown()
+	for(var/datum/subsystem/ss in subsystems)
+		ss.Shutdown()
+
```

Comment:
you could change processing to FALSE in this proc if you wanted to shutdown the mc as well.

not that it will matter much.